x-pack/legacy/plugins/console_extensions/spec/overrides/ml.estimate_memory_usage.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
I haven't been able to use the "__scope_link" feature to point to the `put_data_frame_analytics` object. So, for now, I copied it over.